docs/tutorial/dynamic_loading/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _sec-dynamic-loading:
+
+======================================
+Dynamically loading the shared library
+======================================
+
+In this section, you will find an example of how to call the `xnvme_enumerate` function when dynamically loading the **xNVMe** shared library. The example serves to illustrate how one can go about doing this and some common pitfalls to watch out for. The example will first be presented in C and then in python.
+
+To get an overview of what the **xNVMe** shared library contains refer to :ref:`sec-c-api`
+
+Using the shared library in C
+-----------------------------
```

Comment:
rst: For the "overline" then it needs a space before and over the text:
```
========================================
 Dynamically loading the shared library
========================================
```
It is not required, but it is then convenient that the next under-line style is the same character:
```
foo
===
```
So using them like:
```
=========
 Example
=========

Lorem ... ipsum

bar
===

Lorem ... ipsum

baz
---

Lorem ... ipsum
```

Reply:
fixed in commit 7dbc5d0